moped-database/metadata/tables.yaml

```diff
@@ -3206,6 +3209,53 @@
           - is_deleted
         filter: {}
         check: {}
+  event_triggers:
+    - name: activity_log_moped_project_tags
+      definition:
+        delete:
+          columns: '*'
+        enable_manual: false
+        insert:
+          columns: '*'
+        update:
+          columns:
+            - is_deleted
+            - tag_id
+            - id
+            - project_id
+      retry_conf:
+        interval_sec: 10
+        num_retries: 0
+        timeout_sec: 60
+      webhook_from_env: HASURA_ENDPOINT
+      headers:
+        - name: x-hasura-admin-secret
+          value_from_env: ACTIVITY_LOG_API_SECRET
```

Comment:
I don't think this is documented, but hasura will throw an error if you try to reference any  `HASURA_GRAPHQL_` env variables in the event trigger configuration.

Just a guess, but I suppose they're trying to prevent you from sending your admin secret off to some unknown endpoint. In our case, we're feeding the secret back to our own API, so I don't think this is a concern. It does create the awkward situation where we have the redundant var `ACTIVITY_LOG_API_SECRET` which is fact the `HASURA_GRAPHQL_ADMIN_SECRET`.

Reply:
as an alternative—we _could_ use the JWT token from a session variable. the problem there is that any events that are triggered outside of a session (e.g. a session-less python backfill script) will fail.

i think the simplest approach is to do what we see here 🤷‍♂️